data-raw/stress_test_arguments.R

```diff
@@ -2,15 +2,15 @@
 stress_test_arguments <- tibble::tribble(
                       ~name,       ~type, ~default,               ~allowed,       ~min,       ~max,
                "asset_type", "character",       NA, "equity, bonds, loans",         NA,         NA,
-        "baseline_scenario", "character", "WEO2020_SPS", "WEO2020_SPS, GECO2019_ref",         NA,         NA,
-           "shock_scenario", "character", "WEO2020_SDS", "WEO2020_SDS, GECO2019_1.5c, GECO2019_2c_m",         NA,         NA,
+        "baseline_scenario", "character", "WEO2021_STEPS", "WEO2021_STEPS, GECO2019_ref, WEO2021_APS",         NA,         NA,
+           "shock_scenario", "character", "WEO2021_SDS", "WEO2021_SDS, WEO2021_NZE_2050, GECO2019_1.5c, GECO2019_2c_m",         NA,         NA,
                       "lgd",    "double",   "0.45",                     NA,      "0.3",      "0.9",
            "risk_free_rate",    "double",   "0.02",                     NA,        "0",     "0.05",
             "discount_rate",    "double",   "0.07",                     NA,    "0.015",      "0.1",
               "growth_rate",    "double",   "0.03",                     NA,     "0.01",    "0.099",
   "div_netprofit_prop_coef",    "double",      "1",                     NA,      "0.8",        "1",
                "shock_year",    "double",   "2030",                     NA,     "2025",     "2035",
-       "scenario_geography", "character", "Global", "AdvancedEconomies, Africa, AsiaPacific, Brazil, CentralandSouthAmerica, China, DevelopingEconomies, Emergingmarket&developingeconomies, EU, Eurasia, Europe, Global, India, Japan, LatinAmerica, MiddleEast, NonOECD, NorthAmerica, OECD, Russia, SouthAfrica, SoutheastAsia, US",         NA,         NA,
+       "scenario_geography", "character", "Global", "Africa, AsiaPacific, Brazil, China, EU, Eurasia, Europe, Global, India, Japan, LatinAmerica, MiddleEast, NonOECD, NorthAmerica, OECD, Russia, US",         NA,         NA,
```

Comment:
deleting all the regions that are not in the scenario_geography of the production data